cli/command-reference/roxctl-scanner.adoc

```diff
@@ -6,7 +6,10 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-Commands related to the Scanner service.
+Commands related to the StackRox Scanner and Scanner V4 (Technology Preview) services.
```

Comment:
I was going to comment about IBM style not liking abbreviating "V" for version: https://www.ibm.com/docs/en/ibm-style?topic=references-product-versions#version-release-and-modification-identifiers

I can see now that it's already been used throughout the docs, so it's fine to keep it for consistency. But just something to consider for the future.

Reply:
Yeah, this is the "official" name for it, in the code and in the GUI, so I think I have to follow that.